app/controllers/bookings_controller.rb

```diff
@@ -1,6 +1,10 @@
 class BookingsController < ApplicationController
   def index
     @bookings = Booking.all.ordered
+    per_page = params[:per_page] ? params[:per_page].to_i : 4
+    
+    @bookings = Booking.paginate(page: params[:page], per_page:4)
+    @bookings = Booking.paginate(page: params[:page], per_page: per_page)
```

Comment:
Én igyekszem mindig kerülni, hogy egyszer használatos változóba szervezzek ki kódokat. Csak olyankor szoktam, ha segíti a kód megértését. A kérdőjel kettőspont operátort tudod kicsit kompaktabban is írni:
```ruby
változó.presence || 4
# ugyan azt csinálja, mint
változó.present? ? változó : 'érték'
```
És így elférne egy sorban a kifejezés.

Reply:
Itt a `per_page` -et tök szépen megírtad, viszont az eredeti `@bookings = Booking.all.ordered` benne maradt a kódban.